### PR TITLE
Endpoints refactor

### PR DESCRIPTION
**Pull request by Oxalisviolacea**
Refactor endpoints

- [x] Updated Files:
Added db.py
Added models.py
- [ ] Routes Added:

- [ ] Sad Path Tested

- [ ] Happy Path Tested

- [ ] All Tests are passing

- [ ] Updated README.md with added information

- [x] Collaborators:
@evilaspaas1 @Todd-Estes @PhilipDeFraties

### CODE REVIEW

**Review by evilaspaas1 — APPROVED** [review]
Looks good.

**Review by PhilipDeFraties — APPROVED** [review]
I <3 refactoring

File: resources/routes.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+from .report import ReportsApi, ReportApi
+
+
+def initialize_routes(api):
+ api.add_resource(ReportsApi, '/api/v1/reports')
+ api.add_resource(ReportApi, '/api/v1/reports/<id>')
```
Comment (Todd-Estes) [inline resources/routes.py:7]: A routes file!

File: resources/report.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from flask import Response, request
+from database.models import Report
+from flask_restful import Resource
+
```
Comment (Todd-Estes) [inline resources/report.py:4]: Like a controller!

File: app.py [hunk 3]
```diff
@@ -1,35 +1,16 @@
...
-  body = request.get_json()
-  report = Report(**body).save()
-  id = report.id
-  return {'id': str(id)}, 200
-
-@app.route('/reports/<id>')
-def get_report(id):
-    reports = Report.objects.get(id=id).to_json()
-    return Response(reports, mimetype="application/json", status=200)
+initialize_routes(api)
```
Comment (Todd-Estes) [inline app.py:14]: This is basically a runner file...we'll have to drop it eventually

**Review by Todd-Estes — APPROVED** [review]

**Pull request merged**